BHoM_UI/Components/Engine/External.cs

```diff
@@ -1,4 +1,4 @@
-/*
+﻿/*
```

Comment:
I really don't know why github sometimes defends this belief that a totally different file, which has been removed, has been moved.